src/cluster.c

```diff
@@ -1857,10 +1857,10 @@ void ShardGroupGet(RedisRaftCtx *rr, RedisModuleCtx *ctx)
         node_count++;
         RedisModule_ReplyWithArray(ctx, 2);
 
-        char node_id[RAFT_SHARDGROUP_NODEID_LEN + 1];
+        char node_id[128];
```

Comment:
why are we going to a magic number here?

Reply:
Compiler generates a false positive warning on my local that buffer size might not be enough. Changed it to `RAFT_SHARDGROUP_NODEID_LEN * 2`, I guess it is better.